Исправлена ошибка с отправкой сообщений от бота

gagpa opened the pull request:
### Что сделано
- Убраны "<>". Бот думал, что это теги
- Оформление вызовов метода отправки сообщений пользователю

gagpa committed:
Исправлена ошибка с отправкой сообщений от бота

spamelick commented on app/bot/__init__.py:
```diff
@@ -11,12 +10,9 @@ class ExtensionBot(Bot):
 
     async def send_message(self, *args, **kwargs) -> types.Message:
         kwargs['parse_mode'] = 'html'
-        try:
```
Почему было удалено исключение?

gagpa committed:
Исправлена ошибка с заблокированными пользователями
- Добавлен Makefile
- Добавлен перехват ошибок на заблокированных пользователей
- Добавлено поле - признак, что пользватель заблокировал бота

spamelick commented on app/bot/__init__.py:
```diff
@@ -62,17 +48,19 @@ async def wrapper(message: types.Message):
-                                '/del <Номер записи> - Удалить запись\n'
-                                '/hide - отчистить историю сообщений\n'
-                           )
+    await bot.send_message(chat_id=283782903, text='123')
```
Удалить дебаг-строку.

spamelick commented on app/db.py:
```diff
@@ -26,6 +26,8 @@ class TelegramUser(Base):
     id = sql.Column(sql.Integer, primary_key=True)
     username = sql.Column(sql.VARCHAR(255), nullable=True)
+
+    is_able = sql.Column(sql.Boolean, default=True, nullable=False)
```
https://translate.yandex.ru/?lang=en-ru&text=able - перевод не соответствует.

gagpa committed:
В моделе TelegramUser св-во is_able заменно на inactive_at

spamelick commented on Makefile:
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/make
+
+PROJECT_NAME = bn24_mono
```
Неиспользумые переменные.

gagpa committed:
Удалены не используемые переменные
